src/Ticketing/TMS.Ticketing.Domain/Common/KeyValePair.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TMS.Ticketing.Domain.Common;
+
+public sealed class KeyValePair
```

Comment:
why do you need a dedicated type for that? .NET already has KeyValuePair.

Reply:
I agree it's confusing that the name overlaps with built-in c# classes, I will change this to Detail